plugins/evaluation-plugin/core/src/com/intellij/cce/core/TokenProperties.kt

```diff
@@ -63,6 +63,7 @@ class JvmProperties(private val props: TokenProperties) : TokenProperties by pro
       return SimpleTokenProperties.create(tokenType, location) {
         builder.isStatic?.let { put(STATIC, it.toString()) }
         builder.packageName?.let { put(PACKAGE, it) }
+        builder.qualifiedName?.let { put("qualified name", it) }
```

Comment:
Let's use a const field for this string constant. Then static checks can save us from typos.